app/screens/LoginScreen.tsx

```diff
@@ -20,8 +24,27 @@ const Login = () => {
     navigation.navigate("HealthConditions");
   };
 
-  const signIn = ()  => {
+  const signIn = async ()  => {
     navigateToHealthConditions();
```

Comment:
Remove this. When signing in a user does not need to keep entering in their health conditions. It only happens once during registration.